packages/components/organisms/f-content-cards/src/components/cardTemplates/PostOrderCard.vue

```diff
@@ -79,6 +79,18 @@ export default {
         };
     },
 
+    computed: {
+        cardContainerCard () {
+            return {
+                ...this.card,
+                target: {
+                    attribute: '_blank',
```

Comment:
For accessibility reasons, if we open link in a new tab, we need to add visually hidden text "opens in new tab" or something like that for screen readers to announce

Reply:
As discussed - I've added this to a ticket for actioning once we have translations for copy in our international markets.